lib/util/ts_parser.js

```diff
@@ -768,13 +787,25 @@ shaka.util.TsParser = class {
       frameCropBottomOffset = expGolombDecoder.readUnsignedExpGolomb();
     }
 
-    resolution.height = String(((2 - frameMbsOnlyFlag) *
+    videoInfo.height = String(((2 - frameMbsOnlyFlag) *
         (picHeightInMapUnitsMinus1 + 1) * 16) - (frameCropTopOffset * 2) -
         (frameCropBottomOffset * 2));
-    resolution.width = String(((picWidthInMbsMinus1 + 1) * 16) -
+    videoInfo.width = String(((picWidthInMbsMinus1 + 1) * 16) -
         frameCropLeftOffset * 2 - frameCropRightOffset * 2);
+    videoInfo.codec = 'avc1.' + this.toHex_(profileIdc) +
+        this.toHex_(profileCompatibility) + this.toHex_(levelIdc);
+
+    return videoInfo;
+  }
 
-    return resolution;
+  /**
+   * Convert a number to hex
```

Comment:
This is not specific enough, IMO.  It converts a number to hex, but specifically with two digits, left-padded.  It doesn't handle numbers larger than 8 bits.

Reply:
The use I give it is very simple, do you prefer a local function to avoid having this?

Reply:
It's fine, I just think the comment needs to be improved.  Maybe also the name.

Reply:
How about this:

```js
  /**
   * Convert a byte to 2 digits of hex.  (Only handles values 0-255.)
   */
  byteToHex_(x) {
```

Reply:
Done!